app/assets/stylesheets/pages/registration.scss

```diff
@@ -0,0 +1,12 @@
+//popover css used in signup page for client side validation
```

Comment:
I mean its not much necessary but if we use popover for validation messages then the width of the popover must be increased. But i think i will remove this and use [HTMLSelectElement.setCustomValidity()](https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement/setCustomValidity) as suggested by @vitoravelino